README-sycl.md

```diff
@@ -14,17 +14,25 @@
 
 ## Background
 
-SYCL is a higher-level programming model to improve programming productivity on various hardware accelerators—such as CPUs, GPUs, and FPGAs. It is a single-source embedded domain-specific language based on pure C++17.
+**SYCL** is a high-level parallel programming model designed to improve developers productivity writing code across various hardware accelerators such as CPUs, GPUs, and FPGAs. It is a single-source language designed for heterogeneous computing and based on standard C++17.
 
-oneAPI is a specification that is open and standards-based, supporting multiple architecture types including but not limited to GPU, CPU, and FPGA. The spec has both direct programming and API-based programming paradigms.
+**oneAPI** is an open ecosystem and a standard-based specification, supporting multiple architectures including but not limited to intel CPUs, GPUs and FPGAs. The key components of the oneAPI ecosystem include : 
 
-Intel uses the SYCL as direct programming language to support CPU, GPUs and FPGAs.
+- **DPCPP** *(Data Parallel C++)* : The primary oneAPI SYCL implementation, which includes the icpx/icx Compilers.
+- **oneAPI Libraries** : A set of highly optimized libraries targeting multiple domains *(e.g. oneMKL - Math Kernel Library)*.
+- **oneAPI LevelZero** : A high performance low level interface for fine-grained control over intel iGPUs and dGPUs.
+- **Nvidia & AMD Plugins** : These are plugins extending oneAPI's DPCPP support to SYCL on Nvidia and AMD GPU targets. 
 
-To avoid to re-invent the wheel, this code refer other code paths in llama.cpp (like OpenBLAS, cuBLAS, CLBlast). We use a open-source tool [SYCLomatic](https://github.com/oneapi-src/SYCLomatic) (Commercial release [Intel® DPC++ Compatibility Tool](https://www.intel.com/content/www/us/en/developer/tools/oneapi/dpc-compatibility-tool.html)) migrate to SYCL.
+### Llama.cpp + SYCL 
+To avoid re-inventing the wheel, this SYCL "backend" follows the same design found in other llama.cpp BLAS-based paths such as * OpenBLAS, cuBLAS, CLBlast etc..*. The oneAPI's [SYCLomatic](https://github.com/oneapi-src/SYCLomatic) open-source migration tool (Commercial release [Intel® DPC++ Compatibility Tool](https://www.intel.com/content/www/us/en/developer/tools/oneapi/dpc-compatibility-tool.html)) was used for this purpose.
 
-The llama.cpp for SYCL is used to support Intel GPUs.
+The llama.cpp for SYCL is used to support: 
+- Intel GPUs.
+- Nvidia GPUs.
 
-For Intel CPU, recommend to use llama.cpp for X86 (Intel MKL building).
+*Upcoming support : AMD GPUs*.
+
+For **Intel CPUs**, it is recommend to use llama.cpp for [x86](README.md#intel-onemkl) approach.
```

Comment:
```suggestion
For **Intel CPUs**, it is recommended to the use llama.cpp for [x86](README.md#intel-onemkl) approach.
```
Not sure about the "for X approach" either. It seems that backends are referred as "Builds", so maybe "Blas Intel oneMKL" build should be the term used. It's a bit of a nit though, same as above.